### PR TITLE
Less broad system font stack

### PR DESCRIPTION
**Pull request by OrdinaryJellyfish**
**Changes proposed in this pull request:**

Change the `font-family` in `less/common/scaffolding.less` (add `system-ui` to the front and `Ubuntu, Cantarell, Oxygen` to the middle)

**Confirmed**

- [x] Frontend changes: tested on a local Flarum installation.
- [ ] Backend changes: tests are green (run `php vendor/bin/phpunit`).

### CODE REVIEW

File: less/common/scaffolding.less [hunk 1]
```diff
@@ -9,7 +9,7 @@
 body {
   background: @body-bg;
   color: @text-color;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Ubuntu, Cantarell, Oxygen, Roboto, Helvetica, Arial, sans-serif;
```
Comment (franzliedke) [inline less/common/scaffolding.less]: Does anybody understand why GitHub seems to think that the leading dash changed somehow?

Comment (luceos) [reply less/common/scaffolding.less]: Not seeing why. Pretty odd though 🤔

Comment (OrdinaryJellyfish) [reply less/common/scaffolding.less]: Ah, the mysteries of life :smile:

**Comment by franzliedke** [comment]
Thanks @OrdinaryJellyfish !

**Pull request merged**